cfg/validate_test.go

```diff
@@ -28,6 +29,19 @@ func validLogRotateConfig() LogRotateLoggingConfig {
 	}
 }
 
+func validFileCacheConfig(t *testing.T) FileCacheConfig {
+	t.Helper()
+	return FileCacheConfig{
+		CacheFileForRangeRead:    false,
+		DownloadChunkSizeMb:      50,
+		EnableCrc:                false,
+		EnableParallelDownloads:  false,
+		MaxParallelDownloads:     int64(max(16, 2*runtime.NumCPU())),
```

Comment:
You can probably use a saner value here instead.
```suggestion
		MaxParallelDownloads:    3,
```